plugins/rplanners/rrt.h

```diff
@@ -456,6 +457,11 @@ Some python code to display data::\n\
             // extend A
             ExtendType et = TreeA->Extend(_sampleConfig, iConnectedA);
 
+            // Maybe should be able to pass options to CheckCollisionConstraint function in SpatialTree and also use it as a check here!
+            if (et == ET_Failed && IS_DEBUGLEVEL(Level_Verbose)) {
```

Comment:
Instead of `IS_DEBUGLEVEL`, use a flag passed down through `planningoptions`

Reply:
I just realized that the problem with using `planningoptions` is that it's passed by `binpickingmodule`, but `binpickingmodule` does not allow user to define `planningoptions`, so the only way to get this information is to hardcode the value from `binpickingmodule`

Reply:
You can change `binpickingmodule`